### ISSUE TITLE
Topics should be read when the dialogue is opened

### ISSUE THREAD

**Issue by mickym2** [ioBroker/ioBroker.node-red#535]
I do not know - why this topic is always closed or in stale mode.

A new created state is NOT listed in the dialogue - until the adapter is restarted.

![image](https://github.com/user-attachments/assets/71dcd157-0974-42a5-8bd6-f2aada98a576)

![image](https://github.com/user-attachments/assets/b68af392-2841-4cd3-b063-98a6b4bd855d)

_Originally posted by @mickym2 in https://github.com/ioBroker/ioBroker.node-red/issues/276#issuecomment-2564417265_

**Comment by mickym2** [comment]
Please reproduce it - create a state in admin adapter and the open the dialogue (3 dots ... ) in iobroker in Node etc.

**Comment by Bernd9000** [comment]
Hab das auch mal getestet mit Node-Red Version 6.0.5.
Bei mir wird es sofort aktualisiert.
Hast Du mal in der Browserkonsole nach Fehlern geschaut mit F12 ?
Dort sollte bei einer Änderung der Pfad auftauchen:

![Unbenannt](https://github.com/user-attachments/assets/56d497a6-a9cc-44e1-ad0c-28d9c9ef1a06)

**Comment by mickym2** [comment]
Ok - i will test when 6.0.x is in stable repository.

**Comment by mickym2** [comment]
Still open and not fixed, see here: https://forum.iobroker.net/post/1255692

**Comment by Marc-Berg** [comment]
Yes, the tree is only automatically updated once after the adapter starts when the dialog is opened, but when I click on 'Update tree', all new topics appear.
I don't even want the update to occur every time the dialog is opened, that would take far too long.
